api/rest/restapi.go

```diff
@@ -669,27 +669,74 @@ func (api *API) allocationHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func match(filters []string, status string) bool {
+	for _, filter := range filters {
```

Comment:
> We should convert to TrackerStatus (TrackerStatusFromString()) and compare with the original and not strings.

Can't do this with aliases such as `error` or `queued`

> We will need to match API functionality in the Go API exposed by the main component 

Don't understand this.
Do you mean this code ? https://github.com/ipfs/ipfs-cluster/blob/17599dac639cf6fd70343c72c24b7a8f037c733f/rpc_api.go#L111-L123
and https://github.com/ipfs/ipfs-cluster/blob/17599dac639cf6fd70343c72c24b7a8f037c733f/cluster.go#L753-L763

Reply:
@hsanjuan @lanzafame All comments addressed, except this one

Reply:
I mean https://github.com/ipfs/ipfs-cluster/blob/master/api/types.go#L582.

It would look like (untested):

```diff
diff --git a/api/types.go b/api/types.go
index 1234d09..bc3f426 100644
--- a/api/types.go
+++ b/api/types.go
@@ -34,7 +34,7 @@ var logger = logging.Logger("apitypes")
 // TrackerStatus values
 const (
        // IPFSStatus should never take this value
-       TrackerStatusBug TrackerStatus = iota
+       TrackerStatusBug TrackerStatus = 1 << iota
        // The cluster node is offline or not responding
        TrackerStatusClusterError
        // An error occurred pinning
@@ -60,6 +60,12 @@ const (
        TrackerStatusSharded
 )
 
+// Composite TrackerStatus
+const (
+       TrackerStatusError  = TrackerSTatusClusterError | TrackerStatusPinError | TrackerStatusUnpinError
+       TrackerStatusQueued = TrackerStatusPinQueued | TrackerStatusUnpinQueued
+)
+
 // TrackerStatus represents the status of a tracked Cid in the PinTracker
 type TrackerStatus int
 
@@ -75,6 +81,8 @@ var trackerStatusString = map[TrackerStatus]string{
        TrackerStatusRemote:       "remote",
        TrackerStatusPinQueued:    "pin_queued",
        TrackerStatusUnpinQueued:  "unpin_queued",
+       TrackerStatusError:        "error",
+       TrackerStatusQueued:       "queued",
 }
```